android/src/main/java/org/fossasia/badgemagic/ui/fragments/SavedBadgesFragment.kt

```diff
@@ -1,188 +1,190 @@
-package org.fossasia.badgemagic.ui.fragments
```

Comment:
there are some issue with your changes. rebase your branch to the current development and then make these changes manually, then force push